### DISCUSSION TITLE
Bug: In order to run litestar command I need to install more libraries

### DISCUSSION THREAD

**Discussion by juniozguedes** [litestar-org/litestar#2242] [Q&A] [answered]
### Description

Me as a user wants to run litestar command litestar run
Terminal complains about missing libraries rich, uvicorn and jsbeautifier

### Steps to reproduce

```bash
1. pip install litestar
2. litestar run
```

### Litestar Version

2.x

### Platform

- [X] Linux
- [ ] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

**Comment by provinzkraut** [comment]
How did you find about about the CLI?
If you check out the [documentation page for it](https://docs.litestar.dev/latest/usage/cli.html), the first thing it tells you is that in order to use it, you'll need to install additional dependencies and how to do that.

**Comment by JacobCoffee** [comment, marked as answer]
This is no longer an issue as the CLI is provided in the base package.
